apps/studio/pages/project/[ref]/index.tsx

```diff
@@ -56,7 +59,7 @@ const Home: NextPageWithLayout = () => {
         <div className="flex items-center gap-x-3">
           {project?.status === PROJECT_STATUS.ACTIVE_HEALTHY && <SecurityStatus />}
           {IS_PLATFORM && project?.status === PROJECT_STATUS.ACTIVE_HEALTHY && <ServiceStatus />}
-          {IS_PLATFORM && project?.status === PROJECT_STATUS.ACTIVE_HEALTHY && <Connect />}
+          {!connectDialogUpdate && <Connect />}
```

Comment:
oh! it's because none of the instructions will make sense.

Reply:
> it's because none of the instructions will make sense.

shouldn't we keep it hidden for local / self-host then? (so we keep the IS_PLATFORM check?)

Reply:
verified that this doesn't load for self hosted / local
will reinstate the checks